mrmustard/lab_dev/transformations/transformations.py

```diff
@@ -200,18 +200,18 @@ class Rgate(Unitary):
     def __init__(
         self,
         modes: Sequence[int],
-        theta: Union[float, list[float]] = 0.0,
-        theta_trainable: bool = False,
-        theta_bounds: Tuple[Optional[float], Optional[float]] = (0.0, None),
+        phi: Union[float, list[float]] = 0.0,
+        phi_trainable: bool = False,
+        phi_bounds: Tuple[Optional[float], Optional[float]] = (0.0, None),
```

Comment:
i see there has been some doc-confusion about calling this `phi` vs `theta`, but I'm wondering now if users have been explicitly specifying this as `theta=0.1` (with the kwarg name) because if so, this will break their code. either way, if we really want to rename this set of kwargs, we should change the docstring on lines 182-183 from `theta` to `phi` as well

Reply:
Historically it's always been phi. Sam coded this up using theta, so I reverted it back. I don't think anyone has code written with the new `Rgate` yet for this change to be an issue.